components/ActivityDetails/index.js

```diff
@@ -37,16 +38,21 @@ export default function ActivityDetails({
         ))}
       </StyledCateogriesUl>
       <DeleteButton type="button" onClick={onDelete}>
```

Comment:
Each time you use an icon instead of textContent you should add an aria-label to the button.